### ISSUE TITLE
Why the word embedding for padding symbol is not zero

### ISSUE THREAD

**Issue by MrTuo** [hanxiao/bert-as-service#119]
**Prerequisites**

> Please fill in by replacing `[ ]` with `[x]`.

* [x] Are you running the latest `bert-as-service`?
* [x] Did you follow [the installation](https://github.com/hanxiao/bert-as-service#install) and [the usage](https://github.com/hanxiao/bert-as-service#usage) instructions in `README.md`?
* [x] Did you check the [FAQ list in `README.md`](https://github.com/hanxiao/bert-as-service#faq)?
* [x] Did you perform [a cursory search on existing issues](https://github.com/hanxiao/bert-as-service/issues)?

**System information**

> Some of this information can be collected via [this script](https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh).

- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):
- TensorFlow installed from (source or binary):binary
- TensorFlow version:1.12
- Python version:3.6
- `bert-as-service` version: 1.5.0
- GPU model and memory:P100,16G
- CPU model and memory:

---

### Description

> Please replace `YOUR_SERVER_ARGS` and `YOUR_CLIENT_ARGS` accordingly. You can also write your own description for reproducing the issue.

I'm using this command to start the server:

```bash
bert-serving-start -model_dir ./uncased_L-12_H-768_A-12/ -num_worker=4 -pooling_strategy NONE
```

and calling the server via:
```python
bc = BertClient()
bc.encode()
```

Then this issue shows up:
I'm going to get ELMo-like contextual word embedding and find that the word embedding for padding symbol is not zero. I just wanna know if you used the mask matrix? Should I set it to zero in embedding?
...

**Comment by hanxiao** [comment]
In the current version, if you choose `-pooling_strategy NONE`, then no post-processing (e.g. masking) is performed on the `[batch_size, max_seq_len, num_dim]` output. On the contrary, if you choose a `pooling_strategy`, then a mask is applied *before* pooling to ensure the correctness.

But you are right, the masking behavior should be consistent on all `pooling_strategy` including `NONE`, and though setting padding symbol to zero in `[batch_size, max_seq_len, num_dim]` may not be the perfect solution, it should work in most of the cases.

Will solve this soon.

**Comment by hanxiao** [comment]
fyi, this issue is fixed and available in 1.5.1, please do 
```bash
pip install -U bert-serving-server bert-serving-client
```
